kagami/values-dev.yaml

```diff
@@ -2,7 +2,8 @@ ingress:
   enabled: true
   className: "traefik"
   annotations:
-    traefik.ingress.kubernetes.io/router.middlewares: auth@file
+    traefik.ingress.kubernetes.io/router.middlewares: auth@file,allowCORS@file
```

Comment:
Is it okay that here it's `auth@file,allowCORS@file` but in the other files it's just `auth@file` ? @jgraeger

Reply:
Yeah. auth@file is the middleware for authenticating requests. Thats what we want for this service. The CORS middleware will allow [Cross-Origin-Ressource-Sharing where you can learn more about by clicking this link](https://javascript.info/fetch-crossorigin#why-is-cors-needed-a-brief-history). CORS is actually an important security measure, so I only want to enable it on dev.

Reply:
Got it, Thanks!